templates/slim/helpers.rb

```diff
@@ -3,14 +3,13 @@
 require 'asciidoctor'
 require 'json'
 
+# Needed only in compile-time.
+require 'slim-htag' if defined? Slim
+
 if Gem::Version.new(Asciidoctor::VERSION) <= Gem::Version.new('1.5.3')
```

Comment:
Opal does not implement `Gem`. You can still check the version in JavaScript with:

```js
asciidoctor.getCoreVersion(); // available in the next release of Asciidoctor.js
```

Though you will need to use something like that to compare the version:

https://github.com/asciidoctor/asciidoctor.js/blob/34e64c3e95293a03d459421a55fa34978b971066/spec/share/asciidoctor.spec.js#L3-L7

Reply:
Repeating my above comment: Is that version check even necessary given that the way to use this module is through bundler for ruby (which would take care of making sure proper version of asciidoctor is there)?

Reply:
No, it’s not necessary. I suggest to remove it.